src/Pipeline.php

```diff
@@ -40,115 +47,98 @@ class Pipeline implements Iterator
     /**
      * The current iteration row.
      *
-     * @var \Marquine\Etl\Row
+     * @var \Wizaplace\Etl\Row
      */
     protected $current;
 
     /**
      * The etl extractor.
      *
-     * @var \Marquine\Etl\Extractors\Extractor
+     * @var \Wizaplace\Etl\Extractors\Extractor
      */
     protected $extractor;
 
     /**
      * The array of steps for the pipeline.
      *
-     * @var array
+     * @var Step[]
      */
     protected $steps = [];
 
     /**
      * Set the pipeline extractor.
-     *
-     * @param  \Marquine\Etl\Extractors\Extractor  $extractor
-     * @return void
      */
-    public function extractor(Extractor $extractor)
+    public function extractor(Extractor $extractor): void
     {
         $this->extractor = $extractor;
     }
 
     /**
      * Add a step to the pipeline.
      *
-     * @param  \Marquine\EtlStep  $step
-     * @return void
+     * @param \Wizaplace\Etl\Step $step
      */
-    public function pipe(Step $step)
+    public function pipe(Step $step): void
     {
         $this->steps[] = $step;
     }
 
     /**
      * Set the row limit.
-     *
-     * @param  int  $limit
-     * @return void
      */
-    public function limit($limit)
+    public function limit(int $limit): void
     {
         $this->limit = $limit;
     }
 
     /**
      * Set the number of rows to skip.
-     *
-     * @param  int  $skip
-     * @return void
      */
-    public function skip($skip)
+    public function skip(int $skip): void
     {
         $this->skip = $skip;
     }
 
     /**
      * Get the current element.
      *
-     * @return void
+     * @return string[]|int[]
      */
-    public function current()
+    public function current(): array
     {
         return $this->current->toArray();
     }
 
     /**
      * Move forward to next element.
-     *
-     * @return void
      */
-    public function next()
+    public function next(): void
     {
         $this->key++;
         $this->flow->next();
     }
 
     /**
      * Get the key of the current element.
-     *
-     * @return int
      */
-    public function key()
+    public function key(): int
     {
         return $this->key;
     }
 
     /**
      * Checks if current position is valid.
-     *
-     * @return bool
      */
-    public function valid()
+    public function valid(): bool
     {
-        if (! $this->flow->valid() || $this->limitReached()) {
+        if (!$this->flow->valid() || $this->limitReached()) {
```

Comment:
```suggestion
        if (false === $this->flow->valid() || true === $this->limitReached()) {
```

(hm j'ai pas vérifié ce point sur les fichiers d'avant)